plugins/main/public/react-services/navigation-service.tsx

```diff
@@ -124,6 +124,34 @@ class NavigationService {
     return getCore().application.getUrlForApp(appId, options);
   }
 
+  public switchTab(newTab: any, force: any) {
+    const urlSearchParams = new URLSearchParams(this.getSearch());
+    urlSearchParams.set('tab', newTab);
+    this.navigate(
+      `${this.getPathname()}?${urlSearchParams
+        .entries()
+        .reduce(
+          (total, current) =>
+            (total += `${total ? '&' : ''}${current[0]}=${current[1]}`),
+          '',
+        )}`,
+    );
```

Comment:
`urlSearchParams.entries().reduce` generates a problem since `urlSearchParams.entries()` returns an `Iterable` and `reduce` is an array method. In Google this does not seem to cause a problem, but in Firefox it does. A possible solution may be to leave it as follows:

```typescript
const urlSearchParamsEntries = Array.from(urlSearchParams.entries());
this.navigate(
  `${this.getPathname()}?${urlSearchParamsEntries.reduce(
    (total: string, current: string) =>
      `${total}${total ? '&' : ''}${current[0]}=${current[1]}`,
    '',
  )}`,
    );
```

Reply:
It was decided to make a solution that fits all cases and in one of the methods to use the 'Array from' suggested